jvb/src/main/kotlin/org/jitsi/videobridge/colibri2/Colibri2ConferenceHandler.kt

```diff
@@ -460,3 +481,18 @@ class Colibri2ConferenceHandler(
         }
     }
 }
+
+/**
+ * Return true if "unknown endpoint" and "unknown relay" errors should be ignored when handling [iq]. We ignore errors
+ * if the request simply expires endpoints or a relay (since presumably the entity is already expired), or if it is a
+ * batch update of force-mute state (for the same reason, in order to simplify error handling).
+ */
+private fun shouldIgnoreUnknownEndpoints(iq: ConferenceModifyIQ): Boolean {
+    if (iq.endpoints.any { it.create || it.media.size > 0 || it.transport != null || it.sources != null }) {
+        return false
+    }
+    if (iq.relays.any { !it.expire }) {
```

Comment:
What about relays' included endpoint lists?  It looks like right now unknown endpoints there are silently ignored - is that what we want?

Reply:
I think this is okay, because we only expire them when told to do so in signaling?